pkgs/servers/ankisyncd/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3
+, anki
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ankisyncd";
+  version = "2.1.0";
+  src = fetchFromGitHub {
+    owner = "tsudoko";
```

Comment:
@tsudoko Have you tried to ask for getting in @ankicommunity, so that it would be possible to have the more legitimate-looking url here?

This concern is not blocking merging though, as internet appears to confirm that the ankicommunity one is dead for the time being and yours is the current one.